src/main/java/duke/DukeMan.java

```diff
@@ -0,0 +1,109 @@
+package duke;
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * DukeMan is a new update from the Duke program previously. It manages the tasks of the user.
+ *
+ * @author Dominic Siew Zhen Yu
+ */
+
+public class DukeMan {
```

Comment:
Perhaps you can name your class name in a way that explains what this class is supposed to do, as per the naming guidelines